Makefile

```diff
@@ -360,7 +360,13 @@ CLUSTER_UI_JS := pkg/ui/cluster-ui/dist/main.js
 
 .SECONDARY: $(CLUSTER_UI_JS)
 $(CLUSTER_UI_JS): $(shell find pkg/ui/workspaces/cluster-ui/src -type f | sed 's/ /\\ /g') pkg/ui/yarn.installed pkg/ui/workspaces/db-console/src/js/protos.d.ts | bin/.submodules-initialized
+	$(NODE_RUN) -C pkg/ui/workspaces/cluster-ui yarn build --db-console-target
+
+# package-cluster-ui target can be used to build cluster-ui package for publishing to NPM registry
+.PHONY: package-cluster-ui
+package-cluster-ui: $(shell find pkg/ui/workspaces/cluster-ui/src -type f | sed 's/ /\\ /g') pkg/ui/yarn.installed pkg/ui/workspaces/db-console/src/js/protos.d.ts
 	$(NODE_RUN) -C pkg/ui/workspaces/cluster-ui yarn build
```

Comment:
`cluster-ui` package can be built without `--db-console-target` in case this package will be published to NPM registry.